zer0-os/zOS
# Prompt create account

### What does this do?
![Screenshot 2022-12-01 at 08 44 09](https://user-images.githubusercontent.com/102802882/204995493-5a7e2ef3-30b9-4da1-b9ff-23e95c45bfb2.png)

### Why are we making this change?
Use can change the display name and image profile.

### How do I test this?
- disconnect all wallets that connected to the website - in the same browser
- create a new address
- connect
Expected : you should see the prompt

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
